server/src/main/java/com/networknt/server/Server.java

```diff
@@ -538,4 +538,28 @@ private static void unzipFile(String path, String target) {
             logger.error("IOException", e);
         }
     }
+
+    // method used to merge status.yml and app-status.yml
+    protected static void mergeStatusConfig() {
+        Map<String, Object> appStatusConfig = Config.getInstance().getJsonMapConfigNoCache(STATUS_CONFIG_NAME[1]);
+        if (appStatusConfig == null) {
+            return;
+        }
+        Map<String, Object> statusConfig = Config.getInstance().getJsonMapConfig(STATUS_CONFIG_NAME[0]);
+        List<String> repeatStatusList = new ArrayList<>();
+        for (String key : appStatusConfig.keySet()) {
+            if (statusConfig.containsKey(key)) {
+                repeatStatusList.add(key);
+                if (logger.isInfoEnabled()) {
```

Comment:
I think this must be `logger.isErrorEnabled()`?

Reply:
I saw this during the review but didn't raise it as the logging statement is for an error. We want to output the error regardless logging level and info is lower than the error. If we want to make any change here, I would recommend removing this condition check as we are dealing with errors and server will be shut down. Performance is not a concern here. @jiachen1120 @lanphan What do you think?

Reply:
Sure! I think the condition check is redundant.